src/components/GridLayout.js

```diff
@@ -0,0 +1,118 @@
+//import '../App.css';
+import dynamic from 'next/dynamic';
+import Image from 'next/image';
+import {useState} from 'react';
+
+//import {Suspense} from 'react';
+
+import useThemeStore from '../hooks/useThemeStore';
+//import useThemeStorePersist from '../hooks/useThemeStorePersist';
+//TODO als svg einbinden
+import brightnessDay from '../images/brightness-day.png';
+import brightnessNight from '../images/brightness-night.png';
+import closeBox from '../images/close-box.png';
+import crossHairsOff from '../images/MapButtons_2020_off.png';
+//import crossHairsOn from '../images/MapButtons_2020_on.png';
+
+//import Map from './Map';
+
+const Map = dynamic(() => import('./Map'), {
+	ssr: false,
+	suspense: true,
+});
+
+export default function GridLayout() {
```

Comment:
I want to keep this because its really not the map container. Its really the grid container that holds ActionBar and Map and control their size. A future option is to add a third element to the grid.